meshnets/utils/callbacks.py

```diff
@@ -51,6 +51,32 @@ def finalize(self, status: str) -> None:
 class GPUUsage(Callback):
     """Track and log GPU usage."""
 
+    def __init__(self, log_freq=50):
```

Comment:
Add the class definition comment a better description and difference from this one to the one below.

Reply:
Done, let me know if this is better